src/style.css

```diff
@@ -0,0 +1,132 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@1,200&family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --color-blue: #00acdc;
+  --color-dark: #060b35;
+  --color-white: #fff;
+  --color-light-grey: #616070;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400px;
+}
+
+.card {
+  flex-direction: column;
```

Comment:
This property doesn't work, as display is block here, not flex
![image](https://user-images.githubusercontent.com/96409563/227461030-b84769de-cc1c-481c-a460-096062e9558c.png)